### DISCUSSION TITLE
Flip book functionality

### DISCUSSION THREAD

**Discussion by larmarange** [quarto-dev/quarto-cli#2268] [Q&A] [answered]
Is there any plan for implementing a feature similar to flipbookr allowing to display code and corresponding result step by step?

**Comment by mcanouil** [comment, marked as answer]
I don't think so, especially because it has to work also for Julia, Python, not only R.
But on the same idea: https://github.com/quarto-dev/quarto-cli/discussions/2094#discussioncomment-3465685
